### PR TITLE
Icons update, streamlined design, file size optimization

### PR DESCRIPTION
**Pull request by Sundown3867**
Optimization of icons, stripping of unused geometry, updated design, fixes of graphical errorrs (ie size arrows icons). Overview of all icons is here

Overview_20230827

### CODE REVIEW

File: Data/icons/map_castle.svg
Comment (lordfolken) [inline Data/icons/map_castle.svg:1]: This one seems a lighter color, than the other icons, is there a reason?

File: Data/icons/map_power_plant.svg
Comment (lordfolken) [inline Data/icons/map_power_plant.svg:1]: This one looks cut off?

**Comment by lordfolken** [comment]
Thank you for doing this. The result looks professional and consistent. (There are 1-2 questions above) 

The main issue i see is your git history. When you git pull please do a ``git pull --rebase``
that way your commits are applied on top, and there is no merges.

File: Data/icons/map_castle.svg
Comment (Sundown3867) [reply Data/icons/map_castle.svg:1]: Probably a mistake on my side, fixed

File: Data/icons/map_power_plant.svg
Comment (Sundown3867) [reply Data/icons/map_power_plant.svg:1]: That was designed that way on purpose to evoke a lightning, have also a "standard "version 
obrazek

Comment (Sundown3867) [reply Data/icons/map_power_plant.svg:1]: Thank you for making a review. As always I am bad with github stuff so sorry for mess on my side. I will make some further mainly technical changes to further optimize the size of files and then probably send you all icons in zipped files here

Comment (Sundown3867) [reply Data/icons/map_power_plant.svg:1]: Another varion of powerplant version
obrazek

**Comment by Sundown3867** [comment]
I will close this pull request and do it again better

**Comment by Sundown3867** [comment]
I will close this pull request and do it again better

**Pull request closed without merging**